user/caribou/template.py

```diff
@@ -0,0 +1,47 @@
+pkgname = "caribou"
+pkgver = "0.4.21"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--disable-gtk2-module"]
+hostmakedepends = [
+    "automake",
+    "gettext-devel",
+    "gobject-introspection",
+    "intltool",
+    "pkgconf",
+    "slibtool",
+    "xsltproc",
+]
+makedepends = [
+    "at-spi2-core-devel",
+    "clutter-devel",
+    "glib-devel",
+    "gtk+3-devel",
+    "libgee-devel",
+    "libx11-devel",
+    "libxklavier-devel",
+    "libxml2-devel",
+    "libxtst-devel",
+    "python-gobject-devel",
+]
+depends = ["clutter"]
+pkgdesc = "Text entry and UI navigation application"
+maintainer = "Earldridge Jazzed Pineda <earldridgejazzedpineda@gmail.com>"
+license = "LGPL-2.1-or-later"
+url = "https://wiki.gnome.org/Projects/Caribou"
+source = f"https://download.gnome.org/sources/caribou/{pkgver[:-3]}/caribou-{pkgver}.tar.xz"
+sha256 = "9c43d9f4bd30f4fea7f780d4e8b14f7589107c52e9cb6bd202bd0d1c2064de55"
+tool_flags = {"CFLAGS": ["-Wno-incompatible-function-pointer-types"]}
```

Comment:
```suggestion
# vala
tool_flags = {"CFLAGS": ["-Wno-incompatible-function-pointer-types"]}
```

Reply:
What is that comment for?

Reply:
Wno-incompatible-function-pointer-types is an error that should be fixed

..except in the case of vala where the code it generates is broken in that way, so the comment says that this is intended since we know vala does that. otherwise it just looks like silencing something that has to be fixed.

Reply:
Caribou is indeed written in Vala.